classes/event/entry_created.php

```diff
@@ -42,4 +43,13 @@ public function init() {
         $this->data['objecttable'] = 'cpdlogbook_entries';
     }
 
+    public function get_description() {
+        $a = new \stdClass();
+        $a->userid = $this->get_data()['userid'];
+        $a->entryid = $this->get_data()['objectid'];
+        $a->name = $this->get_data()['other']['entryname'];
+
+        return get_string('eventcreated', 'mod_cpdlogbook', $a);
```

Comment:
So this is a bit of a gotcha - this is one of the rare exceptions where string should hard coded and not be in language strings. The reason is that lets say that the plugin is removed, or if it is updated and the nature of the event changes, we do not want to retrospectively change how an event from the past looks. It should be 'written in stone' so to speak and immutable so you can trust the logs.

This applies only to the get_description, the get_name is fine. Tons of examples in core:

https://github.com/moodle/moodle/blob/master/lib/classes/event/badge_created.php

Everything else looks great